etl-job/fhir_import_export.py

```diff
@@ -345,13 +400,17 @@ def main():
     print(f"[out] {json.dumps(input_data, separators=(',', ':'))}")
 
     program, project = _get_program_project(input_data)
-    schema = 'https://aced-public.s3.us-west-2.amazonaws.com/aced.json'
+
+    schema = os.getenv('DICTIONARY_URL', None)
+    if schema is None:
+        schema = 'https://aced-public.s3.us-west-2.amazonaws.com/aced-test.json'
+        output['logs'].append(f"DICTIONARY_URL not found in environment using {schema}")
```

Comment:
Yes. Meant to remove it in the other place where it was defined and put it in the main function so that the same schema is being used throughout the entire code run.  Will fix this.